gradle/libs.versions.toml

```diff
@@ -84,7 +84,7 @@ androidx-room-runtime = { module = "androidx.room:room-runtime", version.ref = "
 androidx-security-crypto = { module = "androidx.security:security-crypto", version.ref = "androidXSecurityCrypto" }
 androidx-splashscreen = { module = "androidx.core:core-splashscreen", version.ref = "androidxSplash" }
 androidx-work-runtime-ktx = { module = "androidx.work:work-runtime-ktx", version.ref = "androidxWork" }
-bitwarden-sdk = { module = "com.bitwarden:sdk-android", version.ref = "bitwardenSdk" }
+bitwarden-sdk = { module = "com.bitwarden:sdk-android-temp", version.ref = "bitwardenSdk" }
```

Comment:
<img src="https://media0.giphy.com/media/woPJ4I9nDVvL7s3fMv/giphy.gif"/>

whats the context for this?

Reply:
SDK team is shuffling some things around. We have to migrate off the old package repo so they can delete it and use the name for this new one. Relates to https://bitwarden.atlassian.net/browse/PM-14062 and https://bitwarden.atlassian.net/browse/PM-13370.

Reply:
Updated the PR title and message to mention the reference change. Thanks for reminding me.